app/models/solidus_stripe/payment_intent.rb

```diff
@@ -5,27 +5,36 @@ class PaymentIntent < ApplicationRecord
     belongs_to :order, class_name: 'Spree::Order'
     belongs_to :payment_method, class_name: 'SolidusStripe::PaymentMethod'
 
-    def self.prepare_for_payment(payment, **stripe_intent_options)
+    def self.prepare_for_payment(payment, **create_stripe_options)
```

Comment:
non-blocking nit: `create_stripe_options` → `stripe_creation_options`, just a proposal for your consideration